app/models/miq_ae_yaml_import.rb

```diff
@@ -321,4 +322,9 @@ def update_attributes(domain_obj)
   def domain_locked?(domain_name)
     MiqAeDomain.find_by(:name => domain_name)&.contents_locked? ? true : false
   end
+
+  def git_import_domain_check
```

Comment:
@pkomanek Should this method be called is_system_domain? Is it specific to Git can it be used by other types of import like ZIPP'ed YAML's?

Reply:
@mkanoor method name is changed, thanks. This method is probably GIT specific since zip_file import is not able to import into locked domains(already includes system_domain) and bin/update or Domain reset is able to overwrite them.